src/mme-app/mmeStatsPromClient.cpp

```diff
@@ -70,6 +70,8 @@ mme_msg_rx_nas_esm_response(mme_msg_rx_family.Add({{"interface","nas"},{"msg_typ
 mme_msg_rx_s1ap_init_context_response(mme_msg_rx_family.Add({{"interface","s1ap"},{"msg_type","init_context_response"}})),
 mme_msg_rx_nas_attach_complete(mme_msg_rx_family.Add({{"interface","nas"},{"msg_type","attach_complete"}})),
 mme_msg_rx_nas_detach_request(mme_msg_rx_family.Add({{"interface","nas"},{"msg_type","detach_request"}})),
+mme_msg_rx_nas_act_ded_br_ctxt_acpt(mme_msg_rx_family.Add({{"interface","nas"},{"msg_type","act_ded_br_ctxt_acpt"}})),
```

Comment:
I need to give you prometheus json file...Dont handcode in this file . @thakurajayL